Assets/Mirror/Runtime/NetworkServer.cs

```diff
@@ -102,16 +101,8 @@ internal static bool InternalListen(string ipAddress, ushort serverPort, int max
             {
                 s_ServerPort = serverPort;
 
-                if (useWebSockets)
-                {
-                    Transport.layer.ServerStartWebsockets(ipAddress, serverPort);
-                    s_ServerHostId = 0; // so it doesn't return false
-                }
-                else
-                {
-                    Transport.layer.ServerStart(ipAddress, serverPort);
-                    s_ServerHostId = 0; // so it doesn't return false
-                }
+                NetworkManager.singleton.transport.ServerStart(ipAddress, serverPort);
```

Comment:
Currently the ipAddress is being ignored. 
Not all transports need a port,   and some other transport need more than a port.

Call  `ServerStart()`  instead (with no parameters),  and let the transport use it's properties to start in the right port.

Reply:
no more port.
what do you mean with ip ignored?

Reply:
won't some transports need it?

Reply:
_some_  will need it yes.   Those that need it should have it as a property in the transport (just like port)
Those that don't need it should not receive it.